src/RandomNumberGenerator.md

```diff
@@ -1,17 +1,128 @@
 # Random Number Generator
+
 Generates a random number within a given range
 
 ## Result
+
 ```js
-  randomNumber(1, 100); // 68
+randomNumber(1, 100); // 68
 ```
 
 ## Snip Langauge
-* [JavaScript](#javascript)
+
+- [JavaScript](#javascript)
+- [Cpp](#cpp)
+- [Python](#python)
+- [Java](#java)
+- [C#](#c)
+- [Go](#go)
+- [php](#php)
+- [perl](#perl)
 
 ### JavaScript
+
 ```js
-  function randomNumber(min, max) { 
-      return Math.floor(Math.random() * (max - min-1) + min+1);
-  } 
+function randomNumber(min, max) {
+  return Math.floor(Math.random() * (max - min - 1) + min + 1);
+}
+```
+
+### Cpp
+
+```cpp
+const unsigned long long a = 6364136223846793005ULL;
+const unsigned long long c = 1ULL << 30;
+const unsigned long long m = (1ULL << 62) - 1;
+
+int randomNumber(int min, int max) {
+    static unsigned long long seed = 0;
+
+    seed = (seed == 0) ? time(NULL) : (a * seed + c) & m;
+
+    int random_number = min + (seed >> 31) % (max - min + 1);
+    return random_number;
+}
+```
+
+### Python
+
+```py
+def randomNumber(min_value, max_value, seed=None):
```

Comment:
This function returns a single value every time instead of random, please check

Reply:
it was providing  a random value, but still added some more seed complications.

Reply:
same thing mate
![image](https://github.com/rahulrk-dev/snips/assets/47377566/30bb83fb-b624-4d26-b7dd-0cd9dd5b1dae)